includes/wizards/class-settings.php

```diff
@@ -0,0 +1,211 @@
+<?php
+/**
+ * Newspack's Settings
```

Comment:
I think we should rename this something more descriptive like "Modules" instead of just "Settings". Also, this provides a good model for handling optional modules that we might want to use to handle some other features that could be integrated into the plugin...like Podcasts or other small one-off Newspack plugins.

Reply:
My issue with the word "Settings" is that it can be used to describe pretty much anything in the Newspack dashboard, so I was worried it might cause some confusion amongst publishers. "Modules" is still pretty generic but describes the purpose of the page just a little bit more clearly and separately from the rest of the wizard pages. All that said, this is something we can discuss if we end up needing to add more modules to the plugin. Thanks!